CMakeLists.txt

```diff
@@ -650,8 +652,27 @@ if (BUILD_TESTING)
                     -Wno-unknown-pragmas -Wformat-security -Wno-missing-braces -fvisibility=hidden -Wno-unreachable-code
                     -Wno-unused-but-set-variable)
         endforeach(benchmark)
-
     endif()
+
+    find_package (Python3 COMPONENTS Interpreter Development)
```

Comment:
I wonder if we should gate adding these to the test pool with a config flag? Technically, a customer could be building the library today with testing enabled, and this would impose a lot more dependency requirements to do that.

Reply:
Agreed -- adding this gate will also simplify the changes in open_bsd/free_bsd.